### DISCUSSION TITLE
How to get list of idle queues for long time in RabbitMQ?

### DISCUSSION THREAD

**Discussion by Pramod34** [rabbitmq/rabbitmq-dotnet-client#1485] [Q&A] [answered]
We have a use case in our application, where we need to get the list of idle queues for long time (configured value: example 7 days) from the rabbitmq server. Based on the list of queue, further actions like deleting or any other things will be taken care. I have researched could not find anything to get the idle queues list based on (X) number of days. Can you please let me know how can we achieve this use case?

**Comment by michaelklishin** [comment, marked as answer]
This question has nothing to do with this client.

You cannot list queues using this client. You cannot list queues based on this condition using the HTTP API. You can [use queue TTL](https://rabbitmq.com/ttl.html) (not to be confused with message TTL) but keep in mind what the RabbitMQ's definition of an "unused queue" will be.

There are also other [queue properties related to conditional deletion](https://rabbitmq.com/queues.html).
